docs/cni/kubernetes/AWS.md

```diff
@@ -176,7 +176,12 @@ aws ec2 run-instances \
 #  Include the subnet param above if using a non-default VPC
 ```
 
-Deploy at least one worker node using the following command:
+Now, deploy at least one worker node.
+
+First, edit `cloud-config/node-config.yaml` and replace all instances of `kubernetes-master` with your Master's private
+DNS name.  This can be found in the output of the previous command, or the AWS portal.
+
+Then, run the following command to start a node instance.
 
 ```
 aws ec2 run-instances \
```

Comment:
This turns out to be quite a bit of extra work in the mainline case when users have a default VPC (they need to get the $VPC_ID as well).  I think this should stay as it is.